android/gradle.properties

```diff
@@ -0,0 +1,43 @@
+# Project-wide Gradle settings.
+
+# IDE users:
+# Settings specified in this file will override any Gradle settings
+# configured through the IDE.
+
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+# Default value: -Xmx10248m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+
+# The size of the library demands a large amount of RAM to build. Increase as necessary if you get GC errors
+## linux requires 10G, OSX requires 11G
+org.gradle.jvmargs=-XX:MaxPermSize=512m -Xmx2g
+org.gradle.parallel=true
+org.gradle.caching=true
+
+mavenGroupId         = com.microsoft.graph
+mavenArtifactId      = microsoft-graph
+mavenMajorVersion    = 2
+mavenMinorVersion    = 3
+mavenPatchVersion    = 1
```

Comment:
does this need to point to 0.7.0?

Reply:
thanks for checking this. It doesn't matter, we use this project simply to build on the android platform and check we're not using any API that's not available for that given api level. But we don't use it to publish anything.

Reply:
thanks for the clarification.